README.md

```diff
@@ -60,7 +60,7 @@ If the variable is not set or the loading attempt fails, RestfulGit will then at
 
 | Config parameter                     | Default value     | Description                                                                                                                                         |
 |--------------------------------------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
-| RESTFULGIT_REPO_BASE_PATH            | (none)            | Root path for Git repositories. Note: only repositories immediately under this path are currently supported.                                        |
+| RESTFULGIT_REPO_BASE_PATH            | (none)            | Root path for Git repositories.
```

Comment:
Would it be possible to note that a maximum depth of 5 directories is supported?